packages/amplify-category-auth/src/__tests__/provider-utils/awscloudformation/utils/extract-apple-private-key.test.js

```diff
@@ -0,0 +1,27 @@
+const { extractApplePrivateKey } = require('../../../../provider-utils/awscloudformation/utils/extract-apple-private-key');
+
+describe('When extracting apple private key...', () => {
+  const expectedOutput =
+    'MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgIltgNsTgTfSzUadYiCS0VYtDDMFln/J8i1yJsSIw5g+gCgYIKoZIzj0DAQehRANCAASI8E0L/DhR/mIfTT07v3VwQu6q8I76lgn7kFhT0HvWoLuHKGQFcFkXXCgztgBrprzd419mUChAnKE6y89bWcNw';
+
+  it('it should remove comments', () => {
+    const input =
+      '-----BEGIN PRIVATE KEY-----MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgIltgNsTgTfSzUadYiCS0VYtDDMFln/J8i1yJsSIw5g+gCgYIKoZIzj0DAQehRANCAASI8E0L/DhR/mIfTT07v3VwQu6q8I76lgn7kFhT0HvWoLuHKGQFcFkXXCgztgBrprzd419mUChAnKE6y89bWcNw-----END PRIVATE KEY-----';
```

Comment:
just to be 100% sure, I'm assuming these are not valid keys? Also, are they flagged by gitsecrets?

Reply:
These aren't valid keys. I didn't see a gitsecrets flag when committing. Is that in a git commit hook?

Reply:
To do e2e tests, you need a key but it doesn't have to be valid. If you try to do it with a fake value the CFN stack will fail.